package.json

```diff
@@ -43,6 +44,7 @@
   "license": "MPL-2.0",
   "dependencies": {
     "bip39": "^2.5.0",
+    "browserify": "^16.1.1",
```

Comment:
can you add this as a `devDependency` instead? `npm install --save-dev browserify` would do it

Reply:
Done. Again lots of changes in `package-lock.json` file.